### DISCUSSION TITLE
Backups and Migration

### DISCUSSION THREAD

**Discussion by calumma13** [Nachtzuster/BirdNET-Pi#144] [Q&A]
I’ve been running BirdNET on a pi for a few months with good results, but found that my original SD card is far too small. So I’m keen on upgrading to a larger card and thought it may also be a good opportunity to migrate to this fork. 

I have attempted to migrate previously but found it problematic. The system runs fine after the migration, but if I run update and restart the pi it reverts back to the original fork and all data collected after the migration is lost. I’m assuming this is an issue compounded by the small sd card. 

So I’m considering a fresh install of this fork on a new sd card. 

I’ve backed up the database without problem and can re-enter BirdWeather token etc if needed. Not too concerned about losing previous recordings, but would prefer to keep them if possible. Are there any other files that I should be backing up and restoring?

**Comment by alexbelgium** [comment]
Hi, here is a relevant function that Nachzuster is working on : https://github.com/Nachtzuster/BirdNET-Pi/pull/119
And I would recommend this list of files to backup : 
- Your conf & db files (of course ;-) ) :
    - "$HOME/BirdNET-Pi/birdnet.conf"
    - "$HOME/BirdNET-Pi/scripts/birds.db"
- All .txt files that contains subconfigurations, including (if existing as they are generated on first use) :
    - "$HOME/BirdNET-Pi/scripts/disk_check_exclude.txt"
    - "$HOME/BirdNET-Pi/apprise.txt"
    - "$HOME/BirdNET-Pi/BirdDB.txt"
    - "$HOME/BirdNET-Pi/exclude_species_list.txt"
    - "$HOME/BirdNET-Pi/include_species_list.txt"
    - "$HOME/BirdNET-Pi/whitelisted_species_list.txt"
    - "$HOME/BirdNET-Pi/IdentifiedSoFar.txt"
    - "$HOME/BirdNET-Pi/blacklisted_images.txt"
- All BirdSongs main folders :
    - "$HOME/BirdSongs/Extracted/Charts"
    - "$HOME/BirdSongs/Extracted/By_Date"

**Reply by calumma13** [reply]
Thanks for that. Hoping to get the new installation in place next week.
